nvflare/apis/fl_component.py

```diff
@@ -52,6 +58,14 @@ def fire_event(self, event_type: str, fl_ctx: FLContext):
         self._fire(event_type, fl_ctx)
 
     def fire_fed_event(self, event_type: str, event_data: Shareable, fl_ctx: FLContext, targets=None):
+        """Fires a federation event.
```

Comment:
should we describe what a federation event is here?

Reply:
Thanks for the feedback, I have updated the docstrings